packages/pci-sdk-iframe/src/pages/app/components/Debugger.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { PropsWithChildren, useEffect, useState } from 'react';
+
+type IDebuggerProps = any;
+
+export default function Debugger(props: IDebuggerProps) {
+	const [responses, setResponses] = useState<IResponse[]>([]);
+	const [mountDate] = useState<Date>(() => new Date());
+
+	useEffect(() => {
+		const { fetch: origFetch } = window;
+		(window as Window).fetch = async function fetchWithDebug(input: RequestInfo, init?: RequestInit) {
+			const response = await origFetch(input, init);
+			const res = await response.clone();
+			const serverResponse = await res.json();
+			const body = init?.body;
+
+			const newResponse: IResponse = {
+				body,
+				headers: init?.headers,
+				serverResponse,
+				status: res.status,
+				url: res.url,
+			};
+
+			setResponses((s: IResponse[]) => [...s, newResponse]);
+
+			return response;
+		};
+	}, []);
+
+	return (
+		<aside
+			style={{
+				position: 'fixed',
+				background: 'white',
+				top: 0,
+				left: 0,
+				right: 0,
+				border: '1px solid #ddd',
+				overflow: 'auto',
+				maxHeight: '30vh',
+				fontSize: '9px',
+				backgroundColor: '#333',
+				color: 'white',
+			}}
+		>
+			<div>
+				<div>Debugger mounted at: {mountDate.toISOString()}</div>
+				{responses.reverse().map((r: any, i: number) => (
+					<Response key={i} {...r} />
+				))}
+			</div>
+		</aside>
+	);
+}
+
+interface IResponse {
+	status: number;
+	url: string;
+	body?: BodyInit | null;
+	serverResponse?: {
+		code?: number;
+		message?: string;
+	};
+	headers?: HeadersInit;
+}
+
+function Response(props: PropsWithChildren<IResponse>) {
```

Comment:
Maybe we can extract this to a separate file within `components/` and call it `<ServerResponse />`?